### DISCUSSION TITLE
Mount data

### DISCUSSION THREAD

**Discussion by julianalsemmani** [caprover/caprover#2033] [Q&A] [answered]
If I'm hosting Caprover on my VPS, and set a custom docker image, how can I use persistent storage? So mount the docker container to use storage locally on the VPS.

Since I'm using Pocketbase, and on updates, I would need to use the same storage location.

**Comment by githubsaturn** [comment, marked as answer]
When you're creating a new app there is a checkbox for this. Once you create your app with persistent data, you can go to app config and map directories as needed.

![image](https://github.com/caprover/caprover/assets/10582082/45402a35-9342-456b-b769-c96706a1d8b0)

**Reply by deadcoder0904** [reply]
@githubsaturn do i need to check this box too even if i use `docker-compose.yml` bind mount? an example -> https://github.com/deadcoder0904/easypanel-nextjs-sqlite (i have `./_data` (not visible in public repo) that contains `.sqlite` database file)

i still have to check this box right even with docker?

**Reply by githubsaturn** [reply]
Yes, this option allows you to bind named volumes and also map the host path to container's path
